build-system/tasks/get-zindex/index.js

```diff
@@ -87,11 +87,11 @@ function onFileThrough(file, enc, cb) {
  * @param {function()} cb callback to end the stream
  * @return {!Array<!Array<string>>}
  */
-function createTable(filesData, cb) {
+function createTable(filesData, unusedCb) {
```

Comment:
Remove anything that starts with "unused".

Reply:
Scanned the whole PR, removed all others.